### PR TITLE
Memory leaks master

### PR DESCRIPTION
**Pull request by ekrylovich**
Several possible memory leaks were removed. Please review to prevent silly regress.

### CODE REVIEW

File: daemon/src/org/eclipse/iofog/message_bus/Message.java [hunk 1]
```diff
@@ -640,13 +641,10 @@ private int getLength(String str) {
...
 			dataBaos.writeTo(result);
 			return result.toByteArray();
-		} catch (Exception e) {
-		} finally {
-			try {
-				headerBaos.close();
-				dataBaos.close();
-			} catch (Exception e) {}
+		} catch (IOException exc) {
+			LoggingService.logWarning("Message: ", exc.getMessage());
```
Comment (ilaryionava) [inline daemon/src/org/eclipse/iofog/message_bus/Message.java]: "Message :" make a module name constant ?

Comment (ekrylovich) [reply daemon/src/org/eclipse/iofog/message_bus/Message.java]: This is not a module :-)

File: daemon/src/org/eclipse/iofog/utils/BytesUtil.java [hunk 2]
```diff
@@ -28,12 +30,12 @@
...
-		input.read(tmp, 0, tmp.length);
-		input.read(result, 0, result.length);
-		try {
-			input.close();
-		} catch (IOException e) {}
+		try (ByteArrayInputStream input = new ByteArrayInputStream(src)){
+			input.read(tmp, 0, tmp.length);
+			input.read(result, 0, result.length);
+		} catch (IOException e) {
+			LoggingService.logWarning("BytesUtil: ", e.getMessage());
```
Comment (ilaryionava) [inline daemon/src/org/eclipse/iofog/utils/BytesUtil.java]: same here ?

Comment (ekrylovich) [reply daemon/src/org/eclipse/iofog/utils/BytesUtil.java]: done

File: daemon/src/org/eclipse/iofog/message_bus/Message.java (hunk 1 above)
Comment (ekrylovich) [reply daemon/src/org/eclipse/iofog/message_bus/Message.java]: done

**Pull request merged**